Add organization profile (#1)
ballerina-platform/.github: profile/README.md

## Welcome to Ballerina 👋

<h1 align="center">
  <center><img src="../images/ballerina-logo.png" alt="Ballerina Logo"/></center>
</h1>

[Ballerina](https://ballerina.io/) is a statically typed, open-source cloud-native programming language developed and supported by [WSO2](https://wso2.com/).

> With Ballerina, you could easily develop microservices, API endpoints and integrations,

Reviewer: ```suggestion
With Ballerina, you can easily develop microservices, API endpoints and integrations,
```

and any other application for the cloud. Additionally, Ballerina has all the general-purpose
functionalities you could expect from a modern programming language.

With Ballerina's compiler-level built-in support for widely used data types such as JSON and XML, your focus on handling structured data, network service interactions, and dealing with concurrency is made much more effective. 

Ballerina provides language constructs specifically for consuming and delivering network services. Its abstractions and syntax for concurrency and network interaction have been designed to correspond with sequence diagrams closely. This enables a bidirectional mapping for any Ballerina function between its textual representation in the syntax and its graphical representation as a sequence diagram.

## Get started

> Here you can find resources to get started with Ballerina. 

Reviewer: ```suggestion
You can use the resources below to get started with Ballerina. 
```

* [Set up Ballerina and VSCode extension](https://ballerina.io/learn/install-ballerina/set-up-ballerina/)
* [Learn about the language and the platform](https://ballerina.io/learn/)
* [Refer to the Ballerina by Example](https://ballerina.io/learn/by-example/) 

> * [Try in Ballerina Playground](https://play.ballerina.io/)

Reviewer: ```suggestion
* [Try Ballerina in the Playground](https://play.ballerina.io/)
```

* [Discover more about Ballerina packages in Ballerina Central](https://central.ballerina.io/)

## Contribute to Ballerina

> As an open-source project, Ballerina welcomes contributions from the community. To start contributing, read these [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md) for information on how you should go about contributing to our project.

Reviewer: ```suggestion
As an open-source project, Ballerina welcomes contributions from the community. To start contributing, read the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md) for information on how you should go about contributing to the project.
```

## Report issues and security flaws

Report an issue in the relevant repo out of the GitHub repos listed below. 

>**Tip:** If you are unsure whether you have found a bug, search the existing issues in the corresponding repo on 
> GitHub and raise it in the [Ballerina Slack channel](https://ballerina-platform.slack.com/).
  - Compiler, runtime, or tooling: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo
  - Standard library: <a href="https://github.com/ballerina-platform/ballerina-standard-library/issues">ballerina-standard-library</a> repo
  - Language specification: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo
  - Website: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo
  - Extended library: <a href="https://github.com/ballerina-platform/ballerina-extended-library/issues">ballerina-extended-library</a> repo
  - Security flaw: send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.

## License

> Ballerina code is distributed under [Apache license 2.0](https://github.com/ballerina-platform/ballerina-lang/blob/master/LICENSE).

Reviewer: ```suggestion
Ballerina code is distributed under the [Apache license 2.0](https://github.com/ballerina-platform/ballerina-lang/blob/master/LICENSE).
```

## Useful links

> 📧 The ballerina-dev@googlegroups.com mailing list is for discussing code changes to the Ballerina project.

Reviewer: ```suggestion
📧 The [ballerina-dev@googlegroups.com](https://groups.google.com/g/ballerina-dev?pli=1) mailing list is for discussing code changes to the Ballerina project.
```

Author: Re-worded a bit addressing this as well.
